content/author.yaml

```diff
@@ -21,6 +21,12 @@
   bio: I am a User Experience Designer with a mixed background of industrial design, service design, product design and user experience; currently working as a UX designer at a finacial services company.
   avatar: avatars/rb-avatar.jpeg
   twitter: Becky_Bayliss
+- id: Arthur Ceccotti
+  from: Brighton, UK
+  website: https://www.linkedin.com/in/arthur-ceccotti-0042/
```

Comment:
I know, Sam... I will replace it with tuca.ninja soon!

Reply:
<3